sap-3-tier-marketplace-image-multi-sid-db-md/azuredeploy.json

```diff
@@ -151,263 +163,552 @@
+              "diskSizeGB": 1024
             },
             {
-              "size": 512
+              "lun": 5,
+              "caching": "None",
+              "createOption": "Empty",
+              "diskSizeGB": 512
             }
-          ]
+          ],
+          "scriptArguments": {
+            "Windows": "-luns '0,1,2,3,4#5' -names 'data#log' -paths 'C:\\sql\\data#C:\\sql\\log'  -sizes '100#100'"
+          },
+          "useFastNetwork": false
         }
       },
-      "X-Large < 250.000 SAPS": {
-        "Not HA": {
-          "dbserversize": "Standard_GS5",
-          "dbservercount": 1,
-          "dbserverdisks": [
+      "X-Large": {
+        "HANA": {
+          "vmSize": "Standard_M128s",
+          "disks": [
+            {
+              "lun": 0,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 1024
+            },
+            {
+              "lun": 1,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 1024
+            },
             {
-              "size": 1023
+              "lun": 2,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 1024
             },
             {
-              "size": 1023
+              "lun": 3,
+              "caching": "None",
+              "writeAcceleratorEnabled": "true",
+              "createOption": "Empty",
+              "diskSizeGB": 512
             },
             {
-              "size": 1023
+              "lun": 4,
+              "caching": "None",
+              "writeAcceleratorEnabled": "true",
+              "createOption": "Empty",
+              "diskSizeGB": 512
             },
             {
-              "size": 1023
+              "lun": 5,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 1024
             },
             {
-              "size": 1023
+              "lun": 6,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 64
             },
             {
-              "size": 1023
+              "lun": 7,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 2048
             },
             {
-              "size": 1023
+              "lun": 8,
+              "caching": "ReadOnly",
+              "createOption": "Empty",
+              "diskSizeGB": 2048
             }
-          ]
+          ],
+          "scriptArguments": {
+            "Linux": "-luns '0,1,2#3,4#5#6#7,8' -names 'data#log#shared#usrsap#backup' -paths '/hana/data#/hana/log#/hana/shared#/usr/sap#/hana/backup' -sizes '100#100#100#100#100'"
+          },
+          "useFastNetwork": true
         },
-        "HA": {
-          "dbserversize": "Standard_GS5",
-          "dbservercount": 2,
-          "dbserverdisks": [
+        "SQL": {

```

Comment:
seems like we could simplify this quite a bit by using copy loops...

Reply:
it might be possible to use copy loops now for SQL server but we might want to use Write Accelerator for SQL as well and if we do, the disk configs would look different. I would like to keep it like it is

Reply:
ok, something to take a look at for next time - this is 500 lines of code and could probably be done in < 50...